import.ps1

```diff
@@ -11,6 +11,6 @@ if ($BuildAndPublish -eq $true) {
     dotnet publish -c $Configuration
 }
 
-Import-Module  $PSScriptRoot\src\Powershell\bin\$Configuration\netstandard2.0\publish\Cmf.CustomerPortal.Sdk.Powershell.dll
+Import-Module  $PSScriptRoot\src\Powershell\bin\$Configuration\net8.0\publish\Cmf.CustomerPortal.Sdk.Powershell.dll

```

Comment:
we don't have any condition/restriction that we should continue using the netstandard for something?